internal/utils/utils.go

```diff
@@ -156,6 +156,10 @@ func KubeConfig(kubeConfigPath string, kubeContext string) (*rest.Config, error)
 		return nil, fmt.Errorf("kubernetes configuration load failed: %w", err)
 	}
 
+	// avoid throttling request when some Flux CRDs are not registered
+	cfg.QPS = 50
+	cfg.Burst = 100
```

Comment:
Maybe better to add these as arguments to function so that making a `rootArgs` flags based configuration option will be easier in a next iteration?

Reply:
Why would you want to change them?

Reply:
Because there does not seem to be a consensus on the right values to use, with config values ranges from `100/100` to `1e6/1e6` in various popular projects (prometheus-operator, nginx-controller, etc).

Providing the option to configure the value means that if for some reason K8s (or a specific distro) start to behave differently, users can apply a workaround themselves without having to wait on a fix from our side.